packages/core/utils/src/longRunningProcessTimer.ts

```diff
@@ -1,5 +1,5 @@
 import { red } from "colors";
-import { Logger } from 'embark-logger';
+import { Logger } from 'embark';
```

Comment:
This change and https://github.com/embark-framework/embark/pull/2057/files#diff-fb6fb47a1e3c62a1f838bacba50af184R1 serve the purpose of breaking a circular dependency relationship between `embark-utils` and `embark-logger`.

Reply:
As mentioned in the other comment: I don't think we want to make `Logger` type importable from `embark` when the package that introduces that type is actually `embark-logger`.

Every package should come with their own types and should be imported from there. If circular dep is a problem (which I think hasn't so far) then I'd rather suggest we introduce a new package that resolves the circular dep.

Reply:
See https://github.com/embark-framework/embark/pull/2057#discussion_r349618440.